### DISCUSSION TITLE
Ayush's Resume Review

### DISCUSSION THREAD

**Discussion by geekayushk** [recodehive/Stackoverflow-Analysis#355] [Q&A] [answered]
Please review my resume. I would like to know what areas I can improve on my resume, what my ATS score is, and what I can add to make it stand out.
[Ayush_Resume.pdf](https://github.com/user-attachments/files/16349614/Ayush_Resume.pdf)

**Comment by sanjay-kv** [comment]
[ATS Version of CV (15).docx](https://github.com/user-attachments/files/16357143/ATS.Version.of.CV.15.docx)
[aysush res.pdf](https://github.com/user-attachments/files/16357152/aysush.res.pdf)

**Reply by Hassassin-10** [reply, marked as answer]
Good Work

**Comment by SuBroXz11** [comment]
Bro you resume is very good.. 
But I think you should put a  short summary about yourself at the beginning.

**Comment by RaviSingh733** [comment]
Resume is So Good

**Comment by geekayushk** [comment]
Please review my resume. I would like to know what areas I can improve on my resume, I have updated my experience section. 
[Ayush_Resume.pdf](https://github.com/user-attachments/files/17833816/Ayush_Resume.pdf)

**Comment by DrDead0** [comment]
your resume is good but i think you should add Professional Summary which will tell , about your strengths in 2 - 3 lines

**Comment by Hassassin-10** [comment]
> Please review my resume. I would like to know what areas I can improve on my resume, what my ATS score is, and what I can add to make it stand out. [Ayush_Resume.pdf](https://github.com/user-attachments/files/16349614/Ayush_Resume.pdf)

![Screenshot 2024-12-02 230917](https://github.com/user-attachments/assets/b940c913-2bf6-4f61-a148-f7d6280bfe60)
see that brother

**Comment by Aryanjstar** [comment]
you experience and projects are good, i will suggest you to follow the ATS template.

**Comment by viswanathr73** [comment]
Overall your resume is good.  Please check any  ATS Resume Scanner websites  to get the  ATS score and there you can get  instruction's to improve your resume.

**Comment by ghost** [comment]
nice work

**Comment by sandun-harshana** [comment]
you experience and projects are good,
